apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1001,7 +1001,7 @@ static void fs_libc_dirent_readdir_r(void)
 * @precondition     tc_fs_vfs_mkdir should be passed
 * @postcondition    NA
 */
-static void fs_libc_dirent_telldir(void)
+static void tc_fs_libc_dirent_telldir(void)
```

Comment:
This change is not enough, please also modify at the location where these functions are called